.gitignore

```diff
@@ -1,7 +1,6 @@
 Release*/
 Debug*/
 build/
-include/Util/config.h
```

Comment:
This is an auto generated file for SVF home path and should be ignored.

Reply:
It will be generated into the build directory and is used from there.

https://github.com/SVF-tools/SVF/pull/910/commits/d2ec56b8082aeda9d3851128c3bd740d58d74e3a